### PR TITLE
docs(readme.md) 31

### LINKED ISSUE #31: DOC: Operation instructions for CLI and DS
Dev Story:
As a developer I would like to know the quickest commands to type in my terminal to get up and running after cloning the repo, so that I can start onboarding and exploring in the development environment locally.

As a developer I would like to know which tools are used for:
- package management
- codesharing
- bundling
- publishing .
so that I can start onboarding and exploring in the development environment locally.

### PR DESCRIPTION
**Pull request by usrrname**
# Description

in `Readme.md`: adds tooling and operation instructions, explanation of file structure, clarify key features, add links to other repos in Radius ecosystem
in `packages/cli/Readme.md`: Adds operation instructions

doc #31

Fixes #31 (issue) 

## Type of change

Please delete options that are not relevant.
- [x] This change requires a documentation update

# How Has This Been Tested?
N/A
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,23 +1,56 @@
...
 [rangle.io/radius](https://rangle.io/radius)
 
 [Figma File](https://www.figma.com/file/RqENxZWAzGiEWM7COch1Sc/Radius-Design-Kit)
 
 ## Key Features
 
-- Generate themes from Figma
+Use the CLI to:
+
+- Generate themes from Figma URLs
```
Comment (sumitarora) [inline README.md:45]: Not themes but export tokens.

File: packages/cli/README.md [hunk 2]
```diff
@@ -4,6 +4,14 @@ A modular tool to generate and manage Design Systems and other Accelerated Proje
 
 ## Usage
 
+## Available Commands:
+
+Run CLI without global install to generate a new design system folder :
+
+`npx radius-script init`
```
Comment (sumitarora) [inline packages/cli/README.md:11]: we don't want to mention `radius-script` that is just for testing.

File: README.md [hunk 3]
```diff
@@ -1,23 +1,56 @@
...
+
+See also:
+[Radius-Angular](https://github.com/rangle/radius-angular)
+[Radius](https://github.com/rangle/radius) [for React]
+
+## Create Design System instances with the Radius CLI
 
-The Radius is an opinionated assemblage of open-source tools and libraries that allow you to accelerate your design system. The choice of tools, their composition, and a set of foundational components guides you on how to build a constraint-based system.
+1. Instantiate a design system boilerplate!
+   `npx radius-script init`
```
Comment (sumitarora) [inline README.md:25]: we don't want to mention `radius-script` that is just for testing.

File: packages/cli/README.md (hunk 2 above)
Comment (usrrname) [reply packages/cli/README.md:11]: Darn I thought that was the latest alpha version of the cli!

File: README.md (hunk 3 above)
Comment (usrrname) [reply README.md:25]: I'm thinking about what a dev without the global install can do, as I just heard from Rafae and Doga that you can run everything related to the cli with npm / yarn. Is that an attractive option?

**Review by sumitarora — APPROVED** [review]

**Pull request merged**